new-log-viewer/src/contexts/StateContextProvider.tsx

```diff
@@ -116,13 +133,6 @@ const StateContextProvider = ({children}: StateContextProviderProps) => {
 
     const mainWorkerRef = useRef<null|Worker>(null);
 
-    const mainWorkerPostReq = useCallback(<T extends WORKER_REQ_CODE>(
```

Comment:
I believe we can do the same for `handleMainWorkerResp` and `loadFile` but we be better off focusing on the lint workflow setup in this PR.